backend/src/ee/services/secret-rotation-v2/aws-iam-user-secret/aws-iam-user-secret-rotation-fns.ts

```
@@ -0,0 +1,129 @@
+              .deleteAccessKey({
+                UserName: userName,
+                AccessKeyId: key.AccessKeyId
+              })
+              .promise();
+          }
+          return Promise.resolve();
+        })
+      );
+
+      const activeKey = AccessKeyMetadata.find((k) => k.Status === "Active");
+      if (activeKey && activeKey.AccessKeyId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)